src/extensions/default/bramble/lib/UI.js

```diff
@@ -133,7 +133,7 @@ define(function (require, exports, module) {
         if(fontSize && /\d+px/.test(fontSize)) {
             ViewCommandHandlers.setFontSize(fontSize);
         }
-
+		
```

Comment:
Revert this file change: `git checkout master src/extensions/default/bramble/lib/UI.js `